src/lib/tabs/tab-group.ts

```diff
@@ -275,4 +279,42 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
     this._tabBodyWrapperHeight = this._tabBodyWrapper.nativeElement.clientHeight;
     this._renderer.setStyle(this._tabBodyWrapper.nativeElement, 'height', '');
   }
+
+  /** Body content was swiped left/right */
+  _bodyContentSwiped(event: HammerInput): void {
+    if (this.selectedIndex === null) {
+      this.selectedIndex = 0;
+    }
+
+    if (event.direction === HammerDirection.left || event.direction === HammerDirection.right) {
+      let direction = event.direction;
+      if (this._dir.value === 'rtl') {
+        direction = direction === HammerDirection.left ?
+          HammerDirection.right : HammerDirection.left;
```

Comment:
Overflow indent is +4, but this one can also be broken at a higher syntactic level
```ts
direction =
    direction === HammerDirection.left ? HammerDirection.right : HammerDirection.left;
```